cardano-testnet/test/Test/ShutdownOnSlotSynced.hs

```diff
@@ -49,6 +48,8 @@ hprop_shutdownOnSlotSynced = H.integrationRetryWorkspace 2 "shutdown-on-slot-syn
         }
   TestnetRuntime { bftNodes = node:_ } <- testnet fastTestnetOptions conf
 
+  -- Read from stderr handle and fail if anything is present
```

Comment:
Is this a TODO?  There's no code under here.

Reply:
Left over comment